src/components/StoryCardComponent.vue

```diff
@@ -0,0 +1,46 @@
+<script setup>
+import { ref } from "vue";
+
+const showStory = ref(false);
+
+const props = defineProps({
```

Comment:
maybe I'm just missing it, but is this props const ever used anywhere?

Reply:
I removed it. Thanks for the catch.